mmdet/datasets/coco.py

```diff
@@ -166,7 +166,7 @@ def _parse_ann_info(self, img_info, ann_info):
         else:
             gt_bboxes_ignore = np.zeros((0, 4), dtype=np.float32)
 
-        seg_map = img_info['filename'].replace('jpg', 'png')
+        seg_map = img_info['filename'].split(".")[0] + ".png"
```

Comment:
Thank you for your feedback, you may also need to consider the scenario of `a.b.c.jpg`.

Reply:
just use .rsplit(".", 1)[0]

Reply:
Thank you for your reminder and help, I have modified it.